themes/kien-theme/docroot/_diffs/css/custom.css

```diff
@@ -0,0 +1,395 @@
+/* This file allows you to override default styles in one central location for easier upgrade and maintenance. */
+
+@import "compass";
+@import "mixins";
+
+/*-----------------google fonts---------------*/
+
+@font-face {
+  font-family: 'Lora';
+  font-style: normal;
+  font-weight: 400;
+  src: local('Lora'), local('Lora-Regular'), url(http://fonts.gstatic.com/s/lora/v9/4vqKRIwnQQGUQQh-PnvdMA.woff2) format('woff2');
+}
+@font-face {
+  font-family: 'Lora';
+  font-style: normal;
+  font-weight: 700;
+  src: local('Lora Bold'), local('Lora-Bold'), url(http://fonts.gstatic.com/s/lora/v9/P18Nsu9EiYldSvHIj_0e5w.woff2) format('woff2');
+}
+@font-face {
+  font-family: 'Lora';
+  font-style: italic;
+  font-weight: 400;
+  src: local('Lora Italic'), local('Lora-Italic'), url(http://fonts.gstatic.com/s/lora/v9/_RSiB1sBuflZfa9fxV8cOg.woff2) format('woff2');
+}
+@font-face {
+  font-family: 'Lora';
+  font-style: italic;
+  font-weight: 700;
+  src: local('Lora Bold Italic'), local('Lora-BoldItalic'), url(http://fonts.gstatic.com/s/lora/v9/_IxjUs2lbQSu0MyFEAfa7VtXRa8TVwTICgirnJhmVJw.woff2) format('woff2');
+}
+
+
+/*-----------------general----------------*/
+
+
+.aui {
+
+  /*:focus {
+    outline: 0;
+  }*/
+
+  body {
+    background: #F9F9F9;
+    color: #333;
+    font-family: 'Lora', Helvetica Nue, Arial, Helvetica, sans-serif;
+    font-size: 14px;
+    line-height: 1.6em;
+  }
+
+  a {
+    color: #EC651B;
+    text-decoration: none;
+  }
+
+  a:hover {
+    color: #EC651B;
+    text-decoration: none;
+  }
+
+  p {
+    margin: 0 0 1em;
+    font-family: 'Lora', Helvetica Nue, Arial, Helvetica, sans-serif;
+  }
+
+  strong {
+    font-weight: 700;
+  }
+
+  #wrapper {
+    background:#FFF;
+    border: 1px solid #CCC;
+    margin: 30px auto;
+    padding: 30px 30px 15px;
+    width: 980px;
+    box-shadow:  0 0 20px rgba(192,192,192, 0.4);
+    -moz-box-shadow: 0 0 20px rgba(192,192,192, 0.4);
+    -webkit-box-shadow: 0 0 20px rgba(192,192,192, 0.4);
```

Comment:
We should not need to add vendor prefixes.  If we need them, we can use Compass to achieve this.  http://compass-style.org/reference/compass/css3/box_shadow/